tests/bot/exts/moderation/infraction/test_utils.py

```diff
@@ -139,14 +139,14 @@ async def test_notify_infraction(self, send_private_embed_mock):
                         type="Ban",
                         expires="2020-02-26 09:20 (23 hours and 59 minutes)",
                         reason="No reason provided."
-                    ),
+                    ) + utils.INFRACTION_APPEAL_SERVER_FOOTER,
```

Comment:
Likewise, is there a reason all the tests now have this under the description rather than footer?

Reply:
The message isn't in the footer, so it the tests tested that it was, they would fail.